core/src/main/java/org/elasticsearch/cluster/node/DiscoveryNode.java

```diff
@@ -360,16 +360,16 @@ public int hashCode() {
     public String toString() {
         StringBuilder sb = new StringBuilder();
         if (nodeName.length() > 0) {
-            sb.append('[').append(nodeName).append(']');
+            sb.append('{').append(nodeName).append('}');
```

Comment:
Yes, when an IPv6 address is combined with a port number, you really need them. Its needed to make things unambiguous, its needed to work inside a user's browser, its needed to conform with numerous RFCs on the display of this stuff. If you need more justification, see ones like https://tools.ietf.org/html/rfc5952

On the other hand, the brackets in ES logging provide *NOTHING*. Part of me wanted to remove them across all logging across the board.